lib/store.js

```diff
@@ -68,6 +68,18 @@ class Interface extends Type {
     this.fields = fields
   }
 }
+/**
+ * Type is a method definition
+ */
+class Method extends Type {
+  /*:: fields: FieldMap*/
+
+  constructor(name/*: string*/, attributes/*: Attributes*/, fields/*: FieldMap*/ = {}) {
```

Comment:
I'd write new lines here...

Reply:
If this repository had `prettier` installed, we wouldn't have to care :wink:. The `Union` and `Interface` constructors also have these kind of long lines and the linter does not complain, so I'd like to keep it like this. It looks similar to what the other constructors look like:

![screen shot 2018-05-07 at 16 06 39](https://user-images.githubusercontent.com/482561/39703423-aafd7b62-5210-11e8-81c6-82f8be1f90cc.png)

Reply:
Ya, I know ))) Just I like the old-school 80 lettered rule.